documentation/modules/exploit/multi/http/rails_web_console_v2_code_exec.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+ This module exploits an IP whitelist bypass vulnerability in the developer web console included with Ruby on Rails 4.0.x and 4.1.x. This module will also achieve code execution on Rails 4.2.x if the attack is launched from a whitelisted IP range.
+
+## Verification Steps
+**Prerequisites :**
+```
```

Comment:
I would add a description like "This is an example of installing the software on Kali rolling 2016.2 x64"